plugins/ar_spectral.py

```diff
@@ -51,11 +51,13 @@
 from odemis.acq.stream import UNDEFINED_ROI
 import odemis.gui
 from odemis.gui.conf import get_acqui_conf
+from odemis.gui.plugin import Plugin, AcquisitionDialog
+from odemis.util import executeAsyncTask
 import os.path
 import threading
 import time
+
 import odemis.util.driver as udriver

```

Comment:
Good question... I'm not completely sure. It's automatically done by the code refactoring of Eclipse, so I accept it as it is because it'd be a pain to convince it otherwise. I _think_ its reasoning is that the modules renamed via `as` should be highlighted, so they are placed afterwards.